zerver/lib/events.py

```diff
@@ -256,6 +258,9 @@ def fetch_initial_state_data(
 
         # Send server_timestamp, to match the format of `GET /presence` requests.
         state["server_timestamp"] = time.time()
+        state["presence_history_limit_days_for_web_app"] = (
+            settings.PRESENCE_HISTORY_LIMIT_DAYS_FOR_WEB_APP
+        )
```

Comment:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/presence.20data.20cutoffs/near/1933630

Reply:
If you define this here instead:

```
    # Pass parameters to the client-side JavaScript code.                                               
    # These end up in a JavaScript Object named 'page_params'.                                          
    #                                                                                                   
    # Sync this with home_params_schema in base_page_params.ts.                                         
    page_params: dict[str, object] = dict(                                                              
        page_type="home",                                                                               
        ## Server settings.                                                                             
```

Then you won't be adding it to the API, which seems right if it's a webapp only detail.

Reply:
Oh right, I missed this option, definitely better. Fixed.